docs/Systemd-Activation.md

```diff
@@ -0,0 +1,61 @@
+# Using systemd socket activation
+
+_New in v2.9.0_
```

Comment:
Change this to whatever the next release version number will be...

Reply:
Yeah, let's change this to 2.8.3